setup.py

```diff
@@ -28,11 +28,11 @@
 
 http_client_deps = [
     "httpx~=0.23",
-    "rich~=12.6",
+    "rich~=13.0",
     "click~=8.1",
 ]
 ase_deps = ["ase~=3.22"]
-cif_deps = ["numpy~=1.23"]
+cif_deps = ["numpy~=1.24"]
```

Comment:
I think we should change this optional dep to something less strict so avoid interactions with other reqs that use numpy as a dependency, and also because we are barely using numpy ourselves beyond very simple cases.

```suggestion
cif_deps = ["numpy>=1.23"]
```

is probably fine (see https://numpy.org/neps/nep-0029-deprecation_policy.html#nep29 for more info).